themes/edx.org/lms/templates/dashboard.html

```diff
@@ -193,19 +193,6 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
       </ul>
     </section>
   </section>
-  % if xseries_credentials:
```

Comment:
Since this section is being removed from the dashboard, you should stop passing `xseries_credentials` in the context. I would also like to see the `_get_xseries_credentials` helper moved to a utility module. It doesn't belong in the student views.